src/calculatingBullsAndCows.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+function calculatingBullsAndCows(randomNumber, userNumber) {
```

Comment:
usually verbs are used for function names
```suggestion
function calculateBullsAndCows(randomNumber, userNumber) {
```